data/types/food/nutrition.go

```diff
@@ -6,35 +6,59 @@ import (
 )
 
 type Nutrition struct {
-	Carbohydrates *Carbohydrates `json:"carbohydrates,omitempty" bson:"carbohydrates,omitempty"`
+	Carbohydrate *Carbohydrate `json:"carbohydrate,omitempty" bson:"carbohydrate,omitempty"`
+	Energy       *Energy       `json:"energy,omitempty" bson:"energy,omitempty"`
+	Fat          *Fat          `json:"fat,omitempty" bson:"fat,omitempty"`
+	Protein      *Protein      `json:"protein,omitempty" bson:"protein,omitempty"`
```

Comment:
and Fiber?

Reply:
@jh-bate Dietary fiber is in the Carbohydrate structure.